src/catalog.ts

```diff
@@ -97,7 +102,7 @@ export class Dataset {
 }
 
 export class Organization {
-  constructor(readonly iri: IRI, readonly identifier: string) {}
+  constructor(readonly iri: IRI, readonly abbreviation: string) {}
```

Comment:
Similar to the alternate name of a dataset, the alternate name of an organization in most instances will be an abbreviation. But this is not always the case. For example: the alternate name of `Getty Research Institute` is `Getty` (a condensed version, but not an abbreviation).

Reply:
Renamed to `alternateName` (matching Schema.org).